uwazi/add_cids.py

```diff
@@ -0,0 +1,117 @@
+import os
+import requests
+import json
+import subprocess
+
+UWAZI_ROOT = "/home/makeworld/uwazi"
+CID_METADATA_NAME = "cid"
```

Comment:
Actually, could we change this name to be `SHA256CID` as compared to the `BLAKE3CID` we will want to support later?